TopsortAnalytics/src/main/java/com/topsort/analytics/core/HttpClient.kt

```diff
@@ -0,0 +1,106 @@
+package com.topsort.analytics.core
+
+import java.io.Closeable
+import java.io.IOException
+import java.io.OutputStream
+import java.net.HttpURLConnection
+import java.net.MalformedURLException
+import java.net.URL
+import java.util.zip.GZIPOutputStream
+
+const val LIBRARY_VERSION = 1.0
+
+data class HttpResponse(
+    val code : Int,
+    val message : String,
+){
+    fun isSuccessful() : Boolean {
+        return code in 200..299
```

Comment:
could do startsWith s well

Reply:
`code` returned by the native `HttpURLConnection` is an Int